src/network/networkHandler.ts

```diff
@@ -1054,6 +1055,13 @@ async function handleLoadGameState(payload: {
   underworld.enemiesKilled = loadedGameState.enemiesKilled;
   underworld.activeMods = loadedGameState.activeMods;
   underworld.statCalamities = loadedGameState.statCalamities || [];
+  underworld.allowAchievements = loadedGameState.allowAchievements;
+  Object.assign(GameStatistics.underworldStats, loadedGameState.underworldStatsForSerialization);
+  if (!(underworld.players.length > 1)) {
```

Comment:
This `if` statement is hard to understand at first glance.  How about `underworld.players.length == 1`?